group-generators/helpers/data-providers/big-query/queries.ts

```diff
@@ -67,3 +50,34 @@ export const getERC20HoldersQuery = ({
       ORDER BY address
     `;
 };
+
+export const getERC1155HoldersQuery = (key: string, snapshot?: string) => {
+  return`
+    SELECT * FROM sismo_cache.\`query_${key}\`
+      ${
+        snapshot
+        ? `WHERE block_timestamp <= TIMESTAMP("${snapshot}")`
+        : ""
+      }
+  `;
+};
+
+type ContractQueryArgs = {
+  contractAddress: string;
+  network: SupportedNetwork;
+};
+
+export const getContractTransactionsQuery = ({
```

Comment:
It seems more like a `getTokenTransactionsQuery` no? COntracts does not need to be tokens, they can just handle some logic